ote_cli/ote_cli/utils/tests.py

```diff
@@ -100,6 +100,8 @@ def ote_train_testing(template, root, ote_dir, args):
         "--save-model-to",
         f"{template_work_dir}/trained_{template.model_template_id}",
     ]
+    if "--load-weights" in args:
+        command_line.extend(["--load-weights", f'{os.path.join(ote_dir, args["--load-weights"])}'])
     command_line.extend(args["train_params"])
```

Comment:
I think tests for model training need to adopt --load-weights argument if specified.